### PR TITLE
feat(ScriptData): update version python and jq/sh version

### PR DESCRIPTION
**Pull request by lomasson**
- [x] issue #28 
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/raito/blob/main/CONTRIBUTING.md)
- [x] code change includes tests
- suite of #88

### CODE REVIEW

**Comment by lomasson** [comment]
@m-kus i've update + generate some other blocks

File: tests/blocks/block_0.cairo [hunk 1]
```diff
@@ -1,32 +1,32 @@
-// THIS CODE IS GENERATED BY SCRIPT, DO NOT EDIT IT MANUALLY
```
Comment (m-kus) [inline tests/blocks/block_0.cairo:1]: Can you please keep this header? Sometimes there might be a temptation to do adhoc changes :)

File: tests/blocks.cairo [hunk 2]
```diff
@@ -1,8 +1,13 @@
 pub mod block_0;
-pub mod block_170;
```
Comment (m-kus) [inline tests/blocks.cairo:2]: Were there any issues with this particular block?

File: scripts/data/get_blocks.sh [hunk 3]
```diff
@@ -4,22 +4,25 @@ set -o pipefail;
 
 get_single_block() {
     local block_hash=$1
-    ./scripts/data/get_block.sh "$block_hash"
```
Comment (m-kus) [inline scripts/data/get_blocks.sh:7]: It's a good time to add installation instructions to the README :)
Can you please update the doc to include the section about build dependencies?

File: tests/blocks.cairo (hunk 2 above)
Comment (lomasson) [reply tests/blocks.cairo:2]: Not particularly, I just forgot to take it back (he is now in the blocks). But for the block 481824 i don't think it was optimal to save a 6Mo test file in the git.

**Comment by lomasson** [comment]
@m-kus it should be good

**Review by m-kus — APPROVED** [review]

**Pull request merged**